cmd/oras/discover.go

```diff
@@ -17,12 +17,13 @@ import (
 	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
 	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
+	"github.com/xlab/treeprint"
```

Comment:
This is currently under MIT. Is that a concern or can we take dependency on this? 
/cc @lachie83 @SteveLasker

Reply:
The use of dependencies with permissive licenses in fine in the CNCF. It's copyleft licenses like MPL and GPL that you need to explicitly ask for an exception to be granted. Here is the doc from the CNCF stating that MIT is allowed - https://github.com/cncf/foundation/blob/master/allowed-third-party-license-policy.md#approved-licenses-for-allowlist

Reply:
Also - thanks for paying attention to this @sajayantony! It's not easy.